src/components/ServiceCatalog/ServiceCatalog.tsx

```diff
@@ -358,15 +440,29 @@ const ServiceCatalog = ({ isDisabled }: Props) => {
             onClick={o.onClickFunc}
             title={o.title}
             subtitle={o.subtitle}
+            statusText={provisioningLabel === 'reprovisioning' ? 'reprovision' : 'provision'}
```

Comment:
@mfrancisc again - it may be good to keep these labels in constants as well - especially if it effects logic and is duplicated in many spots

Reply:
should be done in https://github.com/RedHatInsights/developer-sandbox-ui/pull/67/commits/836ef45b9442fd2c459a735a5b09f710e7b1c76e